infrastructure/iac/iac_html.go

```diff
@@ -39,7 +39,8 @@ type TemplateData struct {
 	SeverityIcon template.HTML
 	Description  template.HTML
 	Remediation  template.HTML
-	Path         template.HTML
+	ResourcePath template.HTML
```

Comment:
The "Path" name is a bit ambiguous, in this case the Path refers to where the issue can be found, example: 
"[DocId: 0] > input > spec > template > spec > hostNetwork" .